taskcluster/dotnet-package-opt-base.tyml

```diff
@@ -0,0 +1,95 @@
+$if: 'event.event in build.allowed'
+then:
+  taskId: ${taskcluster.taskId}
+  provisionerId: ${taskcluster.docker.provisionerId}
```

Comment:
this will run with `workerType: ci-decision-task`, which is a linux host running docker-worker, the expected payload is not the same as generic-worker running on windows

You might want to verify against how we do in `taskcluster/test-win-opt-base.tyml` for some of those variables

Reply:
Specifically:
```
  provisionerId: ${taskcluster.docker.provisionerId}
  workerType: ${taskcluster.docker.workerTypeWin}
```

Reply:
@stepkillah Once you fix this it should be fine